application/libraries/MY_Form_validation.php

```diff
@@ -1,10 +1,11 @@
 <?php
 
-class MY_Form_Validation extends CI_Form_Validation
+class MY_Form_validation extends CI_Form_validation
 {
 	function alpha_dash_space($str)
     {
+        echo("<!-- alpha_dash_space $str --!>");
```

Comment:
Is this just left over debug code that was accidentally added?

Reply:
Yep, good spot.